sage_trac/ticket_box.py

```diff
@@ -99,6 +99,13 @@ def filter_stream(self, req, method, filename, stream, data):
 
             filters.append(FILTER_DATE.after(elem))
 
+        gitlab_url = "https://gitlab.com/sagemath/dev/trac/-/pipelines/{}/latest"
+        gitlab_elem = tag.div(
+                tag.a(tag.img(src="https://docs.gitlab.com/assets/images/gitlab-logo.svg", border=0, height=32),
```

Comment:
I think instead of just a generic image you can also make this a badge showing the pipeline status.

Is anyone still running/monitoring patchbots, or could we take out the patchbot statuses?  It will start to get cluttered up there otherwise, unless we maybe put the new badges somewhere else...

Reply:
yes, patchbots are still useful and running. Let us keep them for a moment.

Reply:
I did no manage to find the correct link for the badge. The following works for the develop branch..

https://gitlab.com/sagemath/dev/trac/badges/develop/pipeline.svg